lib/vanagon/platform/rpm/sles.rb

```diff
@@ -10,21 +10,21 @@ class SLES < Vanagon::Platform::RPM
         # @return [Array] A list of commands to add a zypper repo for the build system
         def add_repository(definition)
           definition = URI.parse(definition)
-          if @os_version == '10'
-            flag = 'sa'
-          else
-            flag = 'ar'
-          end
+          flag = if @os_version == '10'
```

Comment:
```
lib/vanagon/platform/rpm/sles.rb:13:11: C: Style/ConditionalAssignment: Use the return of the conditional for variable assignment and comparison.
          if @os_version == '10' ...
          ^^^^^^^^^^^^^^^^^^^^^^
```
http://www.rubydoc.info/gems/rubocop/0.41.2/RuboCop/Cop/Style/ConditionalAssignment describes the cop


It looks like I might be able to set 
```
Style/ConditionalAssignment:
  EnforcedStyle: assign_inside_condition
```
but I'm unsure